src/components/search/curated/css/sanity-loading-data.tsx

```diff
@@ -1,137 +1,55 @@
 import * as React from 'react'
-
-import Image from 'next/image'
-import Card from 'components/pages/home/card'
 import groq from 'groq'
 import {sanityClient} from 'utils/sanity-client'
-import Link from 'next/link'
-import {get, first} from 'lodash'
-import SearchCuratedEssential from '../curated-essential'
 
-const SanityLoadingData = () => {
-  const location = 'css landing page'
+export default function LoadDataFromSanity() {
   const [data, setData] = React.useState<any>()
   React.useEffect(() => {
-    const slug = 'rad-css'
+    const slug = 'curated-css-resources'
     async function run() {
       const data = await sanityClient.fetch(
         groq`
-          *[slug.current == 'rad-css']{
+        *[slug.current == $slug][0]{
```

Comment:
using a variable here isn't buying us much of anything, but it's a step in the right direction. I think we need to consider how we'd query a generic topic page and structure that vs creating a specific resource that represents the page itself.

[groq "outer projections"](https://www.sanity.io/docs/how-queries-work#queries-that-dont-start-with-an-d8cbfd8dd14a) are interesting for this, but I haven't solved the generic query yet.